### PR TITLE
Crawler

### PR DESCRIPTION
**Pull request by AviadPepperi**

### CODE REVIEW

File: endpoints/crawler.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Endpoint from '../endpoint';
+import { CrawlerInput, CrawlerOutput } from '../entities';
+import { PapiClient } from '../papi-client';
+
+export class CrawlerEndpoint extends Endpoint<CrawlerOutput> {
+    constructor(service: PapiClient, protected url: string) {
+        super(service, url);
+    }
+    async crawl(input: CrawlerInput) {
```
Comment (Chasky-H) [inline endpoints/crawler.ts:9]: specify response please

File: entities/index.ts [hunk 2]
```diff
@@ -1,37 +1,41 @@
+export * from './user';
```
Comment (Chasky-H) [inline entities/index.ts:1]: what happened here?

Comment (AviadPepperi) [reply entities/index.ts:1]: sorted by length, so much nicer to look at.

File: endpoints/crawler.ts (hunk 1 above)
Comment (AviadPepperi) [reply endpoints/crawler.ts]: specified

File: endpoints/crawler.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import Endpoint from '../endpoint';
+import { CrawlerInput, CrawlerOutput } from '../entities';
+import { PapiClient } from '../papi-client';
+
+export class CrawlerEndpoint extends Endpoint<CrawlerOutput> {
+    constructor(service: PapiClient, protected url: string) {
+        super(service, url);
+    }
+    async crawl(input: CrawlerInput): Promise<CrawlerOutput> {
```
Comment (Chasky-H) [inline endpoints/crawler.ts:9]: wrong output. This endpoint will return a job.

Comment (AviadPepperi) [reply endpoints/crawler.ts]: fixed to `AddonAPIAsyncResult`

**Review by Chasky-H — APPROVED** [review]

**Pull request merged**